src/Components/Modal.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+export default function Modal({ title, children, dismissable, onclose }) {
+  return (
+    <div>
+      <div style={{ color: "black" }} className="modal">
```

Comment:
If this style is static, then can you move it into css classes?

Reply:
I have done it for all the styles you can have a look.